packages/lit-dev-tools-cjs/src/api-docs/transformer.ts

```diff
@@ -608,7 +608,10 @@ export class ApiDocsTransformer {
   private async makeSourceRelativeToMonorepoRoot(source: SourceReference) {
     source.fileName = pathlib.relative(
       this.config.gitDir,
-      pathlib.resolve(this.config.typedocRoot, source.fileName)
+      pathlib.resolve(
+        pathlib.resolve(this.config.typedocRoot, '..'),
+        source.fileName
+      )
```

Comment:
I also think the comment above this method is incorrect now. The `..` is now needed because the TypeDoc sources are reported relative to the lit.dev root directory, not the packages/ directory.

Reply:
The sources are still reported by TypeDoc relative to `packages/`. E.g., `packages/lit-dev-api/api-data/lit-2/repo/packages/lit-html/development/lit-html.d.ts`.

The strange and subtle change (when bumping TypeDoc version) is that `typedocRoot` has changed from `/Users/spyr/repo/lit.dev` to `/Users/spyr/repo/lit.dev/packages`.

I made an assumption that my change seemed like an internal change which continue to fix the issue documented by the comment. I.e. resolving the `source.fileName` in the way we expect.

Reply:
Excellent suggestion to flatten the pathlib.resolve calls!

Reply:
I think relative to `packages/` would mean `lit-dev-api/api-data/lit-2/repo/packages/lit-html/development/lit-html.d.ts`, which is what I saw when logging the `source.fileName` before the update. After update `source.fileName` includes the `packages/` which is why I say relative to `lit.dev` root.

I have not compared `typedocRoot` before and after update though, and that including `/packages` does explain the need for the `..` to not have double `/package`.

Ultimately the method does what it needs to do so I am happy. :)

Reply:
I see! I read `packages/` literally. 🤦 I totally agree with you. My above comments are wrong. Updated with your suggestion.